### ISSUE TITLE
[Accessibility] add a command to focus the closest comment

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#226090]
Type: <b>Feature Request</b>

CC @meganrogge 

## Repros

1. Open any file containing comment thread

1. In editor, move your cursor to any line having comment

1. Once your cursor is on the line, press ctrl+K ctrl+I to see if the comment tolltip opens

## Current Issue

Screen reader users cannot open comment tooltip on the current line via ctrl+K ctrl+I

## Expected Behavior

ctrl+K ctrl+I needs to work for the comment tooltip just like other tooltips.

VS Code version: Code - Insiders 1.93.0-insider (e2b54301a5745870f6b95d81c91fb3e9557d4f08, 2024-08-20T08:04:15.567Z)
OS version: Windows_NT x64 10.0.22631
Modes:

**Comment by meganrogge** [comment]
@jooyoungseo, the comment widget is not a tooltip. We cannot reuse this keybinding to focus the comment widget because it's possible that line will have hover info.

@alexr00, pls lmk if you disagree.

I do wonder if we should have a focus closest comment widget command/keybinding.

**Comment by alexr00** [comment]
👍It does make sense to have an "Expand and focus comment at current line" command.

**Comment by alexr00** [comment]
The new command is "Focus Comment on Current Line" with no default keybinding.

**Comment by alexr00** [comment]
To verify:
- Install an extension that provides comments
  - GitHub Pull Requests can be used for this
  - Sign in
  - Open the Pull Requests view 
  - Open a file diff for a PR. If there aren't any comments in the file, add a comment on one of the changed lines and collapse the comment
- Put your cursor on a line in a file with a collapsed comment
- Open the command palette and run the command "Focus Comment on Current Line"
- Verify that the comment has focus

**Comment by meganrogge** [comment]
This should probably be added to the comments and/or the editor accessibility help dialog for discoverability.
